firestore.rules

```diff
@@ -1,32 +1,31 @@
 rules_version = '2';
 service cloud.firestore {
 	function notUpdating(field) {
-    return !(field in request.resource.data)
-          || resource.data[field].matches(request.resource.data[field])
+   return !(field in request.resource.data)
+    || resource.data[field] == request.resource.data[field]
```

Comment:
This should have more than just 1 space indentation, right? IIRC a wrapped line gets 2x the regular indentation

Reply:
Yes you are right :) thanks